src/Proto3/Suite/JSONPB/Class.hs

```diff
@@ -167,16 +181,42 @@ parseField :: FromJSONPB a
            => A.Object -> Text -> A.Parser a
 parseField = A.explicitParseField parseJSONPB
 
+instance HasDefault E.Encoding where
+  def       = E.pairs mempty
+  isDefault = E.nullEncoding
+
+instance HasDefault A.Value where
+  def       = A.Null
+  isDefault = (== A.Null)
+
 -- * JSONPB rendering and parsing options
 
 data Options = Options
   { optEmitDefaultValuedFields :: Bool
+  , optEmitInlinedOneof :: Bool
+  -- ^ For compatibility with Go JSONPB.
+  --
+  -- If 'True', the following message will be serialized as:
```

Comment:
Minor nitpick: move the `... will be serialized as:` in between the protobuf message definition and the first example serialization

Reply:
Ok